### PR TITLE
Fix wrong lat long and validate seed data

### PR DESCRIPTION
**Pull request by aileendds**
## Checklist

I have…

- [x] run the application locally (`bin/rails server`) and verified that my changes behave as expected.
- [x] run static code analysis (`bin/rubocop`) and vulnerability scan (`bin/brakeman`) against my changes.
- [x] run the test suite (`bin/rake spec`) and verified that all tests pass.
- [x] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [x] thoroughly outlined below the steps necessary to test my changes.

## Summary of Changes

This pull request…

1. Fixes the bad longitude that caused the search bug
2. Add lat long range validation to the location model
3. Make seeding fail on validation failures instead of failing silently

## Testing

To verify the changes proposed in this pull request…
clone this repo,
git checkout db_validation,
set up development dependencies according to CONTRIBUTING.md, (make sure seeding succeeds)
run bin/rails server, and
load up http://localhost:3000, search for "germany" in locator map, check other data

### CODE REVIEW

File: db/seeds/transportation_offices.json [hunk 1]
```diff
@@ -5364,8 +5364,8 @@
       "postal_code": "77034",
       "country_name": "United States",
       "country_code": "US",
-      "latitude": 29.837692,
-      "longitude": -95351038.0
+      "latitude": 29.609559,
```
Comment (aileendds) [inline db/seeds/transportation_offices.json]: updated both of these based on what i got from entering the address into google maps. the original coordinates were kinda off.

File: app/models/location.rb [hunk 2]
```diff
@@ -3,7 +3,8 @@ class Location < ApplicationRecord
 
   belongs_to :locatable, inverse_of: :location, polymorphic: true
 
-  validates :latitude, :longitude, numericality: true, presence: true
+  validates :latitude, numericality: { greater_than_or_equal_to:  -90, less_than_or_equal_to: 90 }, presence: true
+  validates :longitude, numericality: { greater_than_or_equal_to: -180, less_than_or_equal_to: 180 }, presence: true
```
Comment (jgarber623-gov) [inline app/models/location.rb]: This is _so_ much better. :+1:

File: spec/models/location_spec.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  it 'is not valid with an invalid latitude' do
+    expect(Location.new(latitude: 91, longitude: 180, locatable: TransportationOffice.new(name: 'Office'))).to_not be_valid
+    expect(Location.new(latitude: -91, longitude: 180, locatable: TransportationOffice.new(name: 'Office'))).to_not be_valid
+  end
+
+  it 'is not valid with an invalid longitude' do
+    expect(Location.new(latitude: 90, longitude: 181, locatable: TransportationOffice.new(name: 'Office'))).to_not be_valid
+    expect(Location.new(latitude: 90, longitude: -181, locatable: TransportationOffice.new(name: 'Office'))).to_not be_valid
+  end
+end
```
Comment (jgarber623-gov) [inline spec/models/location_spec.rb]: I believe you could DRY up some of this by stashing the `TransportationOffice` instance into a variable:

```rb
RSpec.describe Location, type: :model do
  let(:transportation_office) { TransportationOffice.new(name: 'Office') }

  it 'is valid with valid coordinates' do
    expect(Location.new(latitude: 90, longitude: 180, locatable: transportation_office)).to be_valid
    # etc. etc. etc.
  end

  # etc. etc. etc.
end
```

File: db/seeds.rb [hunk 4]
```diff
@@ -18,45 +18,45 @@
 tutorials = YAML::load_file(Rails.root.join('db', 'seeds', 'tutorials.yml'))
 
 tutorials.each do |tutorial|
-  record = Tutorial.where(title: tutorial['title']).first_or_create(title: tutorial['title'], display_order: tutorial['display_order'])
+  record = Tutorial.where(title: tutorial['title']).first_or_create!(title: tutorial['title'], display_order: tutorial['display_order'])
```
Comment (jgarber623-gov) [inline db/seeds.rb]: Good call here given how we're using seed data. :+1:

**Pull request merged**